draft-ietf-core-senml.md

```diff
@@ -521,24 +524,27 @@ by the application context.
 
 If either the Base Time or Time value is missing, the missing field is
 considered to have a value of zero. The Base Time and Time values are
-added together to get the time of measurement. A time of zero
-indicates that the sensor does not know the absolute time and the
-measurement was made roughly "now". A negative value is used to
-indicate seconds in the past from roughly "now". 
-
-Positive values up to 2^28 indicate seconds in the future from "now".
-These can be used e.g., for actuation use when the desired change
-should happen in the future but the sender or the receiver does not
-have accurate time available.
-
-Values greater than 2^28 represent an absolute time relative to the
-Unix epoch (1970-01-01T00:00Z in UTC time) and the time is counted
-same way as the Portable Operating System Interface (POSIX) "seconds
-since the epoch" {{TIME_T}}.
-
-Because time values up to 2^28 are used for presenting time relative
+added together to get the time of measurement. 
+
+Values less than 2**28 represent time relative to the current time.
+That is, a time of zero indicates that the sensor does not know the
+absolute time and the measurement was made roughly "now". A negative
+value indicates seconds in the past from roughly "now". Positive
+values up to 2**28 indicate seconds in the future from "now". These
```

Comment:
s/These/The latter/
(make reference clearer)

Reply:
This became "Positive values can be used..."